Assets/HoloToolkit/BuildAndDeploy/Editor/BuildDeployTools.cs

```diff
@@ -325,15 +363,15 @@ public static bool BuildAppxFromSLN(string productName, string msBuildVersion, b
             return true;
         }
 
-        private static void IncrementPackageVersion()
+        private static bool SetPackageVersion(bool increment)
```

Comment:
The new name feels wrong... We cannot actually set a version number, only increment or not...

Reply:
Maybe call it UpdatePackageVersion?

Reply:
We're always setting it no matter what the package version is.

Reply:
Makes sense to me now :)